documentation/internal/Overview-of-Navigation-&-URL-structure.md

```diff
@@ -163,47 +163,41 @@ Example: [https://balsamiq.cloud/scs7giw/ponj59g/rD5AC](https://balsamiq.cloud/s
 
 ---
 
-## `/applications/[appId]?mode=assignment`
+## `/review/[appId]`
 
-### Application assignment mode
+Overview/Start page for reviews associated with specific Application. User will see different content depending on permissions. For Reviewers who have assigned content to review, they will have the option to "Start" review, which will create a Review in the database and re-direct to the specific review page.
 
-The “home page” of a specific application during assignment of reviewers.
+## `/review/[appId]/[reviewId]`
 
-The Supervisor -- with the proper permission to assign reviewers -- can do the assignment of reviewers to one (or more) section when the application is submitted to a new stage (the assignment changes per stage). After assigned the reviewers can review the specific sections.
-
-Example: [https://balsamiq.cloud/scs7giw/ponj59g/r3DC3](https://balsamiq.cloud/scs7giw/ponj59g/r3DC3)
+Specific Review page. Assigned Reviewer(s) can mark questions as "Conform" or "Non-conform" and add comments.
 
 ---
 
-## `/applications/[appId]?mode=consolidation`
-
-### Application consolidation mode
+## `/consolidate/[appId]`
 
-The “home page” of a specific application during consolidation of reviewers.
+Consolidation page. Shows all reviews relevant to current stage. Consolidator can start consolidation.
 
-The Supervisor or Chief user (Consolidator) can select the LOQ (List of questions) to be sent back to the Applicant. Edit the comments or the status of a reviewed question is allowed.
+## `/consolidate/[appId]/[consolidateId]`
 
-On clicking the 'Save changes' button will save it for later and the 'Continue' button will load the Consolidation summary page.
-
-Example: [https://balsamiq.cloud/scs7giw/ponj59g/r5273](https://balsamiq.cloud/scs7giw/ponj59g/r5273)
+Specific Consolidation page. Further details TBC.
 
 ---
 
-## `/applications/[appId]/summary?mode=consolidation`
+## `/applications/[appId]?mode=assignment`
 
-### Application consolidation mode summary page
+_Assigment URLs to be decided_
```

Comment:
Not sure if we need this anymore since this is done in the applications list

Reply:
Yeah, have left it there for now as kind of a To-DO. I'd imagine that more complex assignments (multiple reviewers, etc.) might have to be done through their own page. But will update docs properly when we get to Assignment.